algorithms/ollie/api/swagger/swagger.yaml

```diff
@@ -27,7 +27,7 @@ paths:
         \ in the text by running ollie algorithm on it. The response includes the\
         \ sentence and the instances which contains detail about each instance of\
         \ relationship found. \n"
-      operationId: "getRelationshipsGET"
+      operationId: "getRelationshipsPOST"
```

Comment:
This is nitpicking, but as this isn't a real REST call and more an RPC I suggest to not use the REST terminology and name this `extractRelationships`.